.github/workflows/ci-pr-checks.yaml

```diff
@@ -45,9 +45,9 @@ jobs:
       - name: Run make test
         shell: bash 
         run: |
-          make test
+          make test sidecar-test
```

Comment:
nit: suggest using consistent and qualified target names across scheduler and sidecar and retaining the unqualified targets for cross exe chores.
For example: `test` calls `test-sidecar` and `test-epp`. Same for `build`.
Same naming can be applied to docker files (e.g. `dockerfile-sidecar`) and that means we can use `%` targets for commanlity.

Not critical for this PR, perhaps open an issue?